Tests/DependencyInjection/Compiler/AddSwiftMailerTransportPassTest.php

```diff
@@ -20,13 +21,15 @@
  */
 class AddSwiftMailerTransportPassTest extends TestCase
 {
+    use SetUpTearDownTrait;
```

Comment:
a native solution could be to use methods tagged as `@before` instead, which works since a very long time in PHPUnit.